src/modules/createScore.js

```diff
@@ -0,0 +1,17 @@
+import url from './gameData';
+
+const createScore = async (user, score) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+  });
+  const data = await response.json();
```

Comment:
- [ ] nice use of the async await method, on a side note it is always good to add your API calls inside try-catch block in case an error occurs during your API calls :+1: